src/ObjectPool/src/IPooledObjectPolicy.cs

```diff
@@ -3,10 +3,23 @@
 
 namespace Microsoft.Extensions.ObjectPool
 {
+    /// <summary>
+    /// Represents a policy for managing pooled objects.
+    /// </summary>
+    /// <typeparam name="T">The type of object which is being pooled.</typeparam>
     public interface IPooledObjectPolicy<T>
     {
+        /// <summary>
+        /// Create a <typeparamref name="T"/>.
+        /// </summary>
+        /// <returns>The <typeparamref name="T"/> which was created.</returns>
         T Create();
 
+        /// <summary>
+        /// Put an object in the pool.
+        /// </summary>
+        /// <param name="obj">The object to put in the pool.</param>
```

Comment:
`... to return to the pool`

Reply:
I phrased it this way because it's not clear to me that we have any guarantee that the object given to this method was in the pool in the first place (in which case the method is misnamed too, but that's a different story). Is there such a guarantee?

Reply:
No such guarantee as far as I know. But the method name is `Return`, so I think you want to use that term and avoid introducing a new one. It's fine to add clarifying remarks if needed.